Assets/Scripts/Game/WeaponManager.cs

```diff
@@ -481,7 +481,22 @@ private void MeleeDamage(FPSWeapon weapon)
 
                 // TODO: Use correct damage based on weapon and swing type
                 // Just using fudge values during development
```

Comment:
I've been doing research as well - I think the modifiers are slightly different for Daggerfall. The chart on [this](http://en.uesp.net/wiki/Daggerfall:Combat) page was pulled from the Daggerfall manual - I translated that to +2 diagonal, +0 horizontal, +4 vertical, and -2 stab. If we left stab as -4 it would be impossible to do damage by stabbing with an iron dagger.